lib/utils/exit-handler.js

```diff
@@ -59,7 +57,7 @@ process.on('exit', code => {
 
     // Determine whether to show log file message and why it is
     // being shown since in timing mode we always show the log file message
-    const logMethod = showLogFileError ? 'error' : timing ? 'info' : null
+    const logMethod = showLogFileError ? 'error' : timing ? 'notice' : null
```

Comment:
What's going on here?

Reply:
I mean specifically `info` changing to `notice`.  The nested ternary is its own problem lol.

Reply:
Good catch. This is leftover from getting tests passing which now more closely assert what users actually see. In order to keep this level as info the timing flag should force this log to be shown unless silent is explicitly set I think.

Reply:
Reverted this back to `notice`. The current behavior is that this message is not shown by the `--timing` flag alone and requires `--loglevevl` to be `info` or higher. Keep it the same for now and we can change it later if we want.